### PR TITLE
Fix/1326

### LINKED ISSUE #1326: [Server/Client]: client dont show if git env missing
when deploy the server and client localy without the git local end file, error show:
Cannot return null for non-nullable field AuthorizeAppWithGithubResult.url.

**To Reproduce**
1. start up localy server and client
2. enter Sync with GitHub in app
3. trun on and off the option

**Screenshots**
Screen Shot 2021-04-25 at 14 36 31

### PR DESCRIPTION
**Pull request by tupe12334**
resolve #1326

### CODE REVIEW

File: packages/amplication-client/src/Settings/AuthAppWithGithub.tsx [hunk 1]
```diff
@@ -48,11 +48,16 @@ function AuthAppWithGithub({ app, onDone }: Props) {
     START_AUTH_APP_WITH_GITHUB,
     {
       onCompleted: (data) => {
+        console.log(data);
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Settings/AuthAppWithGithub.tsx:51]: remove 
```suggestion

```

File: packages/amplication-client/src/Settings/AuthAppWithGithub.tsx [hunk 2]
```diff
@@ -48,11 +48,16 @@ function AuthAppWithGithub({ app, onDone }: Props) {
...
+
         openSignInWindow(
           data.startAuthorizeAppWithGithub.url,
           "auth with github"
         );
       },
+      onError: (err) => {
+        // console.log(err);
+      },
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Settings/AuthAppWithGithub.tsx:60]:
```suggestion

```

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
well done. see comments

**Review by tupe12334 — COMMENTED** [review]
got it

**Review by tupe12334 — COMMENTED** [review]
update

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**